src/components/header/loginButton.tsx

```diff
@@ -163,7 +163,9 @@ const LoginButton = () => {
                       {user.title} {user.name}
                     </p>
                     <p className="mt-4">
-                      <NavLink to="/profile">View profile</NavLink>
+                      <Button href={"/profile"} variant="quinary">
+                        View profile
+                      </Button>
```

Comment:
```suggestion
                      <Link to={"/profile"} className="btn btn-quinary">
                        View profile
                      </Link>
```

Since the Button component does not support the React Router and causes a full page reload.

There are a couple of other cases where Button is used with href, can you fix these as well?

Reply:
Should be done